applications/ContactStructuralMechanicsApplication/custom_processes/find_intersected_geometrical_objects_with_obb_for_search_process.h

```diff
@@ -0,0 +1,251 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:             BSD License
+//                                       license: StructuralMechanicsApplication/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+#if !defined(KRATOS_FIND_INTERSECTED_GEOMETRICAL_OBJECTS_WITH_OBB_PROCESS_FOR_SEARCH_H_INCLUDED )
+#define  KRATOS_FIND_INTERSECTED_GEOMETRICAL_OBJECTS_WITH_OBB_PROCESS_FOR_SEARCH_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "processes/find_intersected_geometrical_objects_with_obb_process.h"
+
+namespace Kratos
+{
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/**
+ * @class FindIntersectedGeometricalObjectsWithOBBForSearchProcess
+ * @ingroup ContactStructuralMechanicsApplication
+ * @brief This class is a modification of FindIntersectedGeometricalObjectsWithOBBProcess for contact search
+ * @details Fills the serach set. Only works for Conditions
+ * @author Vicente Mataix Ferrandiz
+*/
+class KRATOS_API(CONTACT_STRUCTURAL_MECHANICS_APPLICATION) FindIntersectedGeometricalObjectsWithOBBForSearchProcess
```

Comment:
I'd call it `FindIntersectedGeometricalObjectsWithOBBContactSearchProcess` instead of  `FindIntersectedGeometricalObjectsWithOBBForSearchProcess`

Reply:
OK

Reply:
Done @rubenzorrilla